classes/core/PKPRequest.inc.php

```diff
@@ -154,24 +151,26 @@ function getIfModifiedSince() {
 
 	/**
 	 * Get the base URL of the request (excluding script).
+	 * @param $allowProtocolRelative boolean True iff protocol-relative URLs are allowed
 	 * @return string
 	 */
-	function getBaseUrl() {
+	function getBaseUrl($allowProtocolRelative = false) {
 		$_this =& PKPRequest::_checkThis();
 
-		if (!isset($_this->_baseUrl)) {
```

Comment:
we are not saving the base url in the object anymore because the same request might need protocol relative equals true and equals false?